packages/client-config/src/client-config-schema/config_types_v2.ts

```diff
@@ -0,0 +1,139 @@
+/**
+ * outputs generated by AWS Amplify (Gen 2) backends
+ */
+export type ConfigTypesV2 = {
+  /**
+   * Version of this schema
+   */
+  _version: '2';
+  /**
+   * Outputs generated from defineAuth
+   */
+  auth?: Auth;
+  /**
+   * Outputs generated from defineAuth
+   */
+  custom?: { [key: string]: any };
```

Comment:
nit
```suggestion
  custom?: { [key: string]: unknown };
```

Reply:
auto-generated. We can perform changes but that becomes cumbersome to remember to always update the code generation (unless we whip out our own codegen)